src/ServicePulse.Host/app/lib/bootstrap-3.3.5/css/bootstrap.css

```diff
@@ -1583,8 +1583,8 @@ pre code {
   overflow-y: scroll;
 }
 .container {
-  padding-right: 0;
-  padding-left: 0;
+  padding-right: 15px;
+  padding-left: 15px;
```

Comment:
@sergioc this is the reason: https://github.com/Particular/ServicePulse/pull/364#discussion_r74415589

I would be happy to discuss it on slack